operator/src/main/java/oracle/kubernetes/operator/WebhookReady.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.io.File;
+import java.io.IOException;
+
+/** This task creates the "readiness" indicator so that Kubernetes knows the Operator is ready. */
+public class WebhookReady {
```

Comment:
Copied comment for copied code 😄

Reply:
Fixed.

Reply:
I don't see that it is fixed. I was suggesting that there could be just one class, shared between the operator and webhook, that checked a .ready path at a common path, something like /probes/.ready.